tests/clickhouse-test

```diff
@@ -222,18 +222,55 @@ def need_retry(args, stdout, stderr, total_time):
     )
 
 
-def get_processlist(args):
+def get_processlist_with_stacktraces(args):
     if args.replicated_database:
         return clickhouse_execute_json(
             args,
             """
-        SELECT materialize((hostName(), tcpPort())) as host, *
-        FROM clusterAllReplicas('test_cluster_database_replicated', system.processes)
-        WHERE query NOT LIKE '%system.processes%'
+        SELECT materialize(hostName() || '::' || tcpPort()::String) as host_port, *
+        -- NOTE: view() here to do JOIN on shards, instead of initiator
+        FROM clusterAllReplicas('test_cluster_database_replicated', view(
+            SELECT
+                groupArray((s.thread_id, arrayStringConcat(arrayMap(
+                    x -> concat(addressToLine(x), '::', demangle(addressToSymbol(x))),
+                    s.trace), '\n') AS stacktrace
+                )) AS stacktraces,
+                p.*
+            FROM system.processes p
+            CROSS JOIN system.stack_trace s
+            WHERE
+                has(p.thread_ids, s.thread_id) AND
```

Comment:
`... FROM system.stack_trace WHERE query_id!='' ...` would be much simpler

Reply:
The idea is not to simply print stack_trace in human readable format, but print stack traces for queries.
So this query will print query + stack traces for all threads involved, while proposed will print only stack trace without information about the query (at least query string and maybe settings also can be useful).

Reply:
> stack traces for all threads involved

The problem is that it will print stacktraces for all threads that were ever involved. For example: https://s3.amazonaws.com/clickhouse-test-reports/38541/9b4da86e2e5cf74a8b5b1205bbabc909e357d654/stress_test__debug__actions_.html: `thread_ids` contains ids of some threads that are not currently processing the query and that have boring stacktraces like https://pastila.nl/?0273f848/08a9b72a94f49dc96cedf091cd70a7c9. Sometimes `thread_ids` contains about 100 ids and most of these threads are not interesting and not related to the query anymore.

> will print only stack trace without information about the query

Let's print stacktrace with `query_id`. Since we already print `system.processes` (that contains `query_id` as well), we will have all information about the query. Or `system.processes JOIN system.stack_trace USING query_id` will be ok too.